event-schema/event_schema.go

```diff
@@ -48,6 +48,18 @@ import (
 	"github.com/rudderlabs/rudder-server/utils/timeutil"
 )
 
+func init() {
+	fmt.Println("Initialized the data structures")
+
+	// Following data structures store events and schemas since last flush
+	updatedEventModels = make(map[string]*EventModelT)
+	updatedSchemaVersions = make(map[string]*SchemaVersionT)
+	offloadedEventModels = make(map[string]map[string]*OffloadedModelT)
+	offloadedSchemaVersions = make(map[string]map[string]*OffloadedSchemaVersionT)
+	archivedEventModels = make(map[string]map[string]*OffloadedModelT)
+	archivedSchemaVersions = make(map[string]map[string]*OffloadedSchemaVersionT)
+}
```

Comment:
`non-actionable` It would be better if those variables are not global, for better readability, ownership and testability. This is beyond the scope of this PR, I just mention it.